.env.example

```
@@ -157,3 +157,5 @@ TWITTER_API_VERSION=2
 DISCORD_INVITE_API_URL=https://discord.com/api/v9/invites/MJyWKkCJ5k?with_counts=true&with_expiration=true
 
 TELESCOPE_ENABLED=false
+
+2FA_ENABLED=false
```

```suggestion
2FA_ENABLED=true
```

so it defaults to enabled